libs/application/templates/driving-license/src/dataProviders/EligibilityProvider.ts

```diff
@@ -53,39 +29,40 @@ export class EligibilityProvider extends BasicDataProvider {
       }
     `
 
-    return this.useGraphqlGateway(query, { drivingLicenseType: 'B' }).then(
-      async (res: Response) => {
-        if (!res.ok) {
-          console.error('failed http request', { res })
-          return Promise.reject({
-            reason: 'Náði ekki sambandi við vefþjónustu',
-          })
-        }
+    const res = await this.useGraphqlGateway(query, { drivingLicenseType: 'B' })
 
-        const response = await res.json()
+    if (!res.ok) {
+      console.error('failed http request', { res })
 
-        if (response.errors) {
-          console.error('response errors', { response })
-          return Promise.reject({ reason: 'Ekki tókst að sækja gögn' })
-        }
+      return Promise.reject({
+        reason: 'Náði ekki sambandi við vefþjónustu',
+      })
+    }
 
-        const eligibility = response.data.drivingLicenseApplicationEligibility
+    //     const eligibility = response.data.drivingLicenseApplicationEligibility
+    //     // REMOVE || true - ONLY FOR TESTING PURPOSES.
+    //     if (eligibility.isEligible || true) {
+    //       return Promise.resolve(eligibility.isEligible)
+    //     } else {
+    //       return Promise.reject({
+    //         reason: extractReason(eligibility.requirements),
+    //       })
+    //     }
+    //   },
+    // )
```

Comment:
Should this comment maybe be removed since we can approach this code if needed via version control?